zbuild.h

```diff
@@ -18,16 +18,11 @@
 #      define STDC99
 #    endif
 #  endif
-#  if __STDC_VERSION__ >= 201112L
```

Comment:
I think removing this should not be a part of this PR, to let the possible discussion happen in a separate PR and to let this PR pass more quickly.

Reply:
OK, done.